webpack.admin.config.js

```diff
@@ -0,0 +1,94 @@
+var webpack = require('webpack');
```

Comment:
Does this file need to be in the root directory? Is it possible to move it into the admin directory?